tmol/score/common/tile_atom_pair_evaluation.hh

```diff
@@ -50,18 +68,19 @@ template <
     tmol::Device D,
     int TILE,
     int nt,
+    int NTERMS,
     typename Real,
     typename Int>
 class InterResBlockEvaluation {
  public:
   template <typename AtomPairFunc>
-  static TMOL_DEVICE_FUNC Real eval_interres_atom_pair(
+  static TMOL_DEVICE_FUNC std::array<Real, NTERMS> eval_interres_atom_pair(
       int tid,
       int start_atom1,
       int start_atom2,
       AtomPairFunc f,
       InterEnergyData<Real> const &inter_dat) {
-    Real score_total = 0;
+    std::array<Real, NTERMS> score_total = {};
```

Comment:
Does this need to be initialized to 0?

Reply:
it does not,  the = {} does that for std:array